eclipse-collections-testutils/src/main/java/org/eclipse/collections/impl/test/Verify.java

```diff
@@ -2242,59 +2240,29 @@ public static void assertEqualsAndHashCode(String itemNames, Object objectA, Obj
     {
         if (objectA == null || objectB == null)
         {
-            fail("Neither item should be null: <" + objectA + "> <" + objectB + '>');
+            Assertions.fail("Neither item should be null: <" + objectA + "> <" + objectB + '>');
         }
 
-        assertFalse("Neither item should equal null", objectA.equals(null));
-        assertFalse("Neither item should equal null", objectB.equals(null));
+        assertFalse(objectA.equals(null), "Neither item should equal null");
+        assertFalse(objectB.equals(null), "Neither item should equal null");
         assertNotEquals("Neither item should equal new Object()", objectA.equals(new Object()));
         assertNotEquals("Neither item should equal new Object()", objectB.equals(new Object()));
-        assertEquals("Expected " + itemNames + " to be equal.", objectA, objectA);
-        assertEquals("Expected " + itemNames + " to be equal.", objectB, objectB);
-        assertEquals("Expected " + itemNames + " to be equal.", objectA, objectB);
-        assertEquals("Expected " + itemNames + " to be equal.", objectB, objectA);
+        assertEquals(objectA, objectA, "Expected " + itemNames + " to be equal.");
+        assertEquals(objectB, objectB, "Expected " + itemNames + " to be equal.");
+        assertEquals(objectA, objectB, "Expected " + itemNames + " to be equal.");
+        assertEquals(objectB, objectA, "Expected " + itemNames + " to be equal.");
         assertEquals(
-                "Expected " + itemNames + " to have the same hashCode().",
                 objectA.hashCode(),
-                objectB.hashCode());
```

Comment:
Newer versions of java don't like this - so time to clean up

Reply:
But this is unrelated to the version of JUnit, isn't it? Where did you see that error? Locally, using newer versions of Java? In our GitHub workflows/actions?